libraries/PDM/src/PDM.cpp

```diff
@@ -145,7 +158,7 @@ int PDMClass::available()
 
   size_t avail = _doubleBuffer.available();
 
-  //NVIC_EnableIRQ(PDM_IRQn);
+  NVIC_EnableIRQ(PDM_IRQn);
```

Comment:
By disabling the PDM in the NVIC, wouldn't this potentially cause the silent loss of samples of data?
Especially with softdevice enabled, and Bluetooth active, which can pre-empt the execution here and cause unexpectedly long delays while this is still disabled?